File: bindings/python/perf/single/perf_stream_len32be.py
#!/usr/bin/env python3
import sys

from perf_common import parse_pattern_args
import perf_stream


def main_from_args(args) -> int:
    parsed = parse_pattern_args("STREAM_LEN32BE", args)
    if parsed is None:
        return 1
    transport, size = parsed
    return perf_stream.run_len32be(transport, size)


if __name__ == "__main__":
    raise SystemExit(main_from_args(sys.argv[1:]))

File: bindings/node/perf/single/perf_stream_len32be.js
'use strict';

const { parsePatternArgs, runStreamLen32Be } = require('./perf_main');

async function main() {
  const parsed = parsePatternArgs('STREAM_LEN32BE', process.argv.slice(2));
  if (!parsed) {
    process.exit(1);
    return;
  }
  const { transport, size } = parsed;
  process.exit(await runStreamLen32Be(transport, size));
}

main().catch(() => process.exit(2));
